commands/command_unlock.go

```diff
@@ -55,6 +62,41 @@ func unlockCommand(cmd *cobra.Command, args []string) {
 	Print("'%s' was unlocked", args[0])
 }
 
+func unlockCheckFileStatus(path string) {
+	modified, err := git.IsFileModified(path)
+
+	if err != nil {
+		Exit(err.Error())
+	}
+
+	if modified {
+		if unlockCmdFlags.Force {
+			// Only a warning
+			Error("Warning: unlocking with uncommitted changes because --force")
+		} else {
+			Exit("Cannot unlock file with uncommitted changes")
+		}
+
+	}
+}
+
+func unlockCheckFileStatusById(id string, lockClient *locking.Client) {
+	// Get the path so we can check the status
+	filter := map[string]string{"id": id}
+	// try local cache first
+	locks, _ := lockClient.SearchLocks(filter, 0, true)
+	if len(locks) == 0 {
```

Comment:
#1884 is specifically about the upcoming `/locks/verify` endpoint. If your server doesn't support LFS, I'd expect running `lfs unlock` to error.